apis/visibility/v1alpha1/conversion_test.go

```diff
@@ -23,13 +23,15 @@ import (
 	"github.com/google/go-cmp/cmp"
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/util/diff"
+	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
 
 	. "sigs.k8s.io/kueue/apis/visibility/v1alpha1"
-	"sigs.k8s.io/kueue/pkg/visibility/api"
 )
 
 func TestPendingWorkloadsOptions(t *testing.T) {
-	codec := runtime.NewParameterCodec(api.Scheme)
+	Scheme := runtime.NewScheme()
+	utilruntime.Must(AddToScheme(Scheme))
```

Comment:
`utilruntime.Must` is just converting an error in a panic, it might be to much for a test, just get the error from `AddToScheme(Scheme)` and t.Fail if not nil.

Reply:
Ok, thanks for the suggestion. Is t.Fatal instead of t.Fail fine? t.Fail marks as failed but continues the execution and in this case I would prefer to stop immediately.

Reply:
sure